internal/blockchain/ethereum/ethereum.go

```diff
@@ -745,9 +745,7 @@ func (e *Ethereum) DeployContract(ctx context.Context, nsOpID, signingKey string
 		"definition": definition,
 		"contract":   contract,
 	}
-	if signingKey != "" {
```

Comment:
This check doesn't make sense since at line 743 we already set "from" to signingKey